include/exadg/aero_acoustic/single_field_solvers/fluid.h

```diff
@@ -57,8 +58,11 @@ class SolverFluid
 
   void
   setup(std::shared_ptr<FluidAeroAcoustic::ApplicationBase<dim, Number>> application,
-        MPI_Comm const                                                   mpi_comm,
-        bool const                                                       is_test)
+        bool const                             compute_acoustic_from_analytical_cfd_solution,
+        std::shared_ptr<dealii::Function<dim>> analytical_cfd_solution_velocity,
+        std::shared_ptr<dealii::Function<dim>> analytical_cfd_solution_pressure,
```

Comment:
According to comment https://github.com/exadg/exadg/pull/659/files#r1599659156, these parameters can be removed in my opinion because they are available via the `pde_operator`.

Reply:
[5efc732](https://github.com/exadg/exadg/pull/659/commits/5efc7327890f5a84d6d550396244124f94633f33)